pkg/capabilities/consensus/ocr3/ocr3.go

```diff
@@ -0,0 +1,22 @@
+package consensus
+
+import (
+	"context"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/types"
+)
+
+type OCR3 struct {
```

Comment:
IMO this could be called OCR3Capability or OCR3ConsensusCapability. It encapsulates all elements needed for the capability to work (interface + plugin + store).

Reply:
Yep, renamed to OCR3Capability 👍